api/cmd/bootstrap_test.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/caraml-dev/mlp/api/pkg/authz/enforcer"
+	enforcerMock "github.com/caraml-dev/mlp/api/pkg/authz/enforcer/mocks"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStartKetoBootsrap(t *testing.T) {
+	tests := []struct {
+		name                               string
+		projectReaders                     []string
+		mlpAdmins                          []string
+		expectedUpdateAuthorizationRequest enforcer.AuthorizationUpdateRequest
+	}{
+		{
+			"admin role must have project post permission",
```

Comment:
We can choose a better name for the two test in order to differentiate the two. Something like:
"admin role must have project creation permission even if there are no members in role"